src/algebra/invertible.lean

```diff
@@ -212,6 +213,9 @@ lemma nonzero_of_invertible [mul_zero_one_class α] (a : α) [nontrivial α] [in
 λ ha, zero_ne_one $ calc   0 = ⅟a * a : by simp [ha]
                          ... = 1 : inv_of_mul_self a
 
+@[priority 100] instance ne_zero.invertible [mul_zero_one_class α] [nontrivial α] (a : α)
```

Comment:
Should it be called `invertible.ne_zero` or `ne_zero.of_invertible`? The name `ne_zero.invertible` suggests that the implication goes in the other direction.